keras_cv/models/object_detection/retinanet/feature_pyramid.py

```diff
@@ -48,5 +48,31 @@ def call(self, inputs, training=False):
         p4_output = self.conv_c4_3x3(p4_output, training=training)
         p5_output = self.conv_c5_3x3(p5_output, training=training)
         p6_output = self.conv_c6_3x3(c5_output, training=training)
-        p7_output = self.conv_c7_3x3(tf.nn.relu(p6_output), training=training)
+        p7_output = self.conv_c7_3x3(ops.relu(p6_output), training=training)
         return p3_output, p4_output, p5_output, p6_output, p7_output
+
+    def build(self, input_shape):
+        p3_channels = input_shape["P3"][-1]
+        p4_channels = input_shape["P4"][-1]
+        p5_channels = input_shape["P5"][-1]
+        self.conv_c3_1x1.build((None, None, None, p3_channels))
+        self.conv_c4_1x1.build((None, None, None, p4_channels))
+        self.conv_c5_1x1.build((None, None, None, p5_channels))
+        self.conv_c3_3x3.build((None, None, None, 256))
+        self.conv_c4_3x3.build((None, None, None, 256))
+        self.conv_c5_3x3.build((None, None, None, 256))
+        self.conv_c6_3x3.build((None, None, None, p5_channels))
+        self.conv_c7_3x3.build((None, None, None, 256))
+        self.built = True
+
+    def compute_output_shape(self, input_shape):
+        p3_shape = input_shape["P3"][:-1]
+        p4_shape = input_shape["P4"][:-1]
+        p5_shape = input_shape["P5"][:-1]
+        return (
+            (p3_shape + (256,)),
```

Comment:
As a defensive measure it's a good idea to cast to tuple what you're trying to concatenate with a tuple (it might be a list or a TensorShape)

Reply:
Done